modules/electrophysiology_browser/README.md

```diff
@@ -41,23 +41,9 @@ electrode information, task event information, the actual recording) -- as well
 
 ## Updating Derivative Files
 
-When annotations are added or edited in the browser, the associated derivative files for this data must also be updated. This is triggered for a specific dataset when a download is selected in the browser. This can also be set up to run automotically for all derivative datasets in your system. 
-To trigger this to run regularly, a script is provided under `tools/update_annotation_files.php` which can be run by cronjob as follows.
-
-In general, the syntax of the cron job command is as follows:   
-`crontab -e 1 2 3 4 5 /path/to/script`     
-where   
-```
-1 2 3 4 5
-| | | | |
-| | | | ----- Day of week (0 - 7) (Sunday=0 or 7)
-| | | ------- Month (1 - 12)
-| | --------- Day of month (1 - 31)
-| ----------- Hour (0 - 23)
-------------- Minute (0 - 59)
-```
-
-For example, this commmand will run the script on the first day of every month at midnight.   
+When annotations are added or edited in the browser, updates are maded in the database but not directly in the associated derivative files for this data. These derivative files must also be updated. When a download is slected in the browser, the derivative files for the selected dataset are updated before downloading is triggered. All derivative files in the file system can be updated using the [update_annotation_files](../../tools/update_annotation_files.php) tool provided. This tool can be triggered to run automatically using a cronjob: 
+
+This commmand will run the script on the first day of every month at midnight.   
```

Comment:
mmm

Reply:
```suggestion
Below is a sample crontab entry to execute this script *every night* at midnight  
```
Please also change the cronjob fields so that it's not first of the month, but every night